### PR TITLE
RDM-10539 - Update befta-fw to latest pre-release version

### PR DESCRIPTION
**Pull request by sullivangareth**
### JIRA link (if applicable) ###

RDM-10539 "Update User Role Column and codebase rename of UserRole to AccessProfile"

### Change description ###

Update User Role Column and codebase rename of UserRole to AccessProfile.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -10,8 +10,8 @@ def component = "manage-case-assignment"
 GradleBuilder builder = new GradleBuilder(this, product)
 
 // Variables to switch pipeline logic and wiring per type of build
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
-def dataStoreApiDevelopPr    = "PR-1260" // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-961"  // This doesn't change frequently, but when it does, only change this value.
```
Comment (smathangi) [inline Jenkinsfile_CNP:13]: I guess we'll re-point to original ones once relevant PRs are merged, right?

**Review by smathangi — APPROVED** [review]

File: Jenkinsfile_CNP (hunk 1 above)
Comment (sullivangareth) [reply Jenkinsfile_CNP:13]: correct

Comment (sullivangareth) [reply Jenkinsfile_CNP]: Fixed in https://github.com/hmcts/aac-manage-case-assignment/pull/294/commits/5c8856254497a08030df10133dd1957ee2f1e6c5

**Review by mattnayler — APPROVED** [review]

**Comment by sullivangareth** [comment]
Closing as this PR has served its purpose in proving that MCA is compatible with befta-fw 7.0.0 release and corresponding definition store changes merged to develop in https://github.com/hmcts/ccd-definition-store-api/pull/961

**Pull request closed without merging**